README.md

```diff
@@ -0,0 +1,40 @@
+<div align="center">
+
+## ForgeFed Interface
+
+**Enabling [software forge](https://en.wikipedia.org/wiki/Forge_(software)) federation in API-space federation**
+
+[![Build](https://github.com/forgefedv2/interface/actions/workflows/linux.yml/badge.svg)](https://github.com/forgefedv2/interface/actions/workflows/linux.yml)
+[![codecov](https://codecov.io/gh/forgefedv2/interface/branch/master/graph/badge.svg?token=0100H4ECG4)](https://codecov.io/gh/forgefedv2/interface)
+
+</div>
+
+## Why
+
+Developing [Free
+Software](https://www.gnu.org/philosophy/free-sw.en.html) is about
+liberating users and giving them total control over how the programs
+they run should work. It's only fair that developers of such software
+enjoy the same levels of liberty. ForgeFed is an attempt to enable
+federation of major software forges(GitLab, GitHub, Gitea, Source Hut,
+etc.) entirely in API-space. Prior to this, [there was an attempt to do
+the same](https://forgefed.peers.community/) but it a very involved
+protocol and required participation from the forge developers. We
+believe our API-space implementation will allow for more organic growth
+as it will require any involvement from the forge developers.
```

Comment:
I might be wrong here, but shouldn't it be, "for more organic growth as it will **not** require any involvement from the forge developers."?

Reply:
good catch, fixed